app/models/post.rb

```diff
@@ -55,12 +55,9 @@ class Post < ApplicationRecord
   has_many :post_follows, dependent: :destroy
   has_many :comments, dependent: :destroy
 
-  scope :in_group, ->(group) { where(target_group: group) }
-  scope :visible_for, ->(user) {
-    joins(:target_group).merge(Group.visible_for(user))
-                        .or(joins(:target_group)
-                        .where(target_group_id: nil))
-  }
+  scope :sfw, (-> { where(nsfw: false) })
+  scope :in_group, (->(group) { where(target_group: group) })
+  scope :visible_for, (->(user) { where(target_group_id: Group.visible_for(user)) })
```

Comment:
Is there a reason for these parens? I mean, they're not hurting anyone, they're just kind of... odd?

Reply:
rubocop was complaining about em. at least i think it was... 🤔  lemme check my editor

Reply:
Also, if I'm not mistaken, aren't you missing a call to add the "or not in group"?

Reply:
yeah rubocop asked for the parens. and, yes, yes i am.